src/theory/datatypes/datatypes_rewriter.cpp

```diff
@@ -221,7 +219,8 @@ RewriteResponse DatatypesRewriter::postRewrite(TNode in)
       }
       else
       {
-        cases.push_back(Node::null());
+        Assert(k == in.getNumChildren() - 1);
```

Comment:
Reading the standard, this case is actually allowed, so maybe we shouldn't have this assertion?

Reply:
Yes, its outdated, will remove.